robotTransform.yaml

```diff
@@ -0,0 +1,12 @@
+%YAML:1.0
+---
+PoseState: !!opencv-matrix
+   rows: 4
+   cols: 4
+   dt: d
+   data: [ 9.9670000000000003e-01, -4.0399999999999998e-02,
+       6.9900000000000004e-02, 5.6950000000000000e+02,
+       2.0799999999999999e-02, 9.6519999999999995e-01,
+       2.6060000000000000e-01, 3.1769999999999999e+02,
+       -7.8000000000000000e-02, -2.5829999999999997e-01,
+       9.6289999999999998e-01, 5.6929999999999995e+02, 0., 0., 0., 1. ]
```

Comment:
Since this is a 4x4 matrix, I think you should structure this so that it also looks like a 4x4 matrix (if possible)

Reply:
data: [ 9.9670000000000003e-01, -4.0399999999999998e-02, 6.9900000000000004e-02, 5.6950000000000000e+02,
       2.0799999999999999e-02, 9.6519999999999995e-01, 2.6060000000000000e-01, 3.1769999999999999e+02,
       -7.8000000000000000e-02, -2.5829999999999997e-01, 9.6289999999999998e-01, 5.6929999999999995e+02, 
       0., 0., 0., 1. ]

Reply:
Or somethng like that

Reply:
I created this yaml file with python from a numpy matrix. Do you think it is still better to modify it? The yaml file that our hand eye outputs also has 2 values per row.

Reply:
Not sure, I just find it a bit annoying to look at.

Reply:
I've checked with Aleksandar and it is how opecv makes it, so defaul. Let's not change it.

Reply:
Ok